ports/esp32/README.md

```diff
@@ -26,34 +26,28 @@ There are two main components that are needed to build the firmware:
   different to the compiler used by the ESP8266)
 - the Espressif IDF (IoT development framework, aka SDK)
 
-Instructions for setting up both of these components are provided by the
-ESP-IDF itself, which is found at https://github.com/espressif/esp-idf .
-Follow the guide "Setting Up ESP-IDF", for Windows, Mac or Linux.  You
-only need to perform up to "Step 2" of the guide, by which stage you
-should have installed the cross-compile and cloned the ESP-IDF repository.
+The ESP-IDF changes quickly and MicroPython only supports a certain version. The
+git hash of this version can be found by running `make` without a configured
+`ESPIDF`. Then you can fetch only the given esp-idf using the following command:
+
+    $ git clone https://github.com/espressif/esp-idf.git
+    $ git checkout <Current supported ESP-IDF commit hash>
+    $ git submodule update --recursive
+
+The binary toolchain (binutils, gcc, etc.) can be installed using the following
+guides:
+
+  * [Linux installation](https://esp-idf.readthedocs.io/en/latest/get-started/linux-setup-scratch.html)
+  * [MacOS installation](https://esp-idf.readthedocs.io/en/latest/get-started/macos-setup-scratch.html)
+  * [Windows installation](https://esp-idf.readthedocs.io/en/latest/get-started/windows-setup-scratch.html)
```

Comment:
Since it's ok to use the precompiled binaries, please remove the `-scratch` part of the URL.